badge.go

```diff
@@ -50,7 +50,12 @@ func demo() {
 }
 
 func badgeProfile() {
+	display.ClearDisplay()
+	showRect(0, 0, WIDTH, 15, black)
+	showRect(0, 15, WIDTH, 15, white)
```

Comment:
Hello Sylr,

Thanks for your contribution, please try this other solution as showRect twice + display.Display() is not needed, **calling display.ClearDisplay() should be enough** as it calls ClearBuffer + Display()